### ISSUE TITLE
Support alfred_ssm helper nested in lists

### ISSUE THREAD

**Issue by gdunkle** [awslabs/aws-control-tower-customizations#43]
Provide support for alfred_ssm helper nested in lists

```
[{
                "ParameterKey": "Key",
                "ParameterValue":[
                        "$[alfred_ssm_/key1]",
                        "$[alfred_ssm_/key2]",
                        "$[alfred_ssm_/key3]"
                    ]
            }]
```

**Comment by groverlalit** [comment]
Hello @gdunkle Thanks for opening this issue and pull request. 
Can you please elaborate on the use case for this feature request? Thanks.

**Comment by gdunkle** [comment]
@groverlalit, I ran into this with a customer who wanted to share out multiple subnets via RAM. Currently they have to deploy the vpc then hardcode the subnet ids in a parameter file. 

Template
```
---
AWSTemplateFormatVersion: '2010-09-09'
Description: Share resources via RAM
Parameters:
    Name:
        Description: The name of the resource share.
        Type: String
    AllowExternalPrincipals:
        AllowedValues:
          - 'true'
          - 'false'
        Default: 'false'
        Description: Indicates whether principals outside your AWS organization can be associated with a resource share. 
        Type: String
    Principals:
        Description: The principals to associate with the resource share. The possible values are IDs of AWS accounts, the ARN of an OU or organization from AWS Organizations. 
        Type: CommaDelimitedList
    ResourceArns:
        Description: The Amazon Resource Names (ARN) of the resources to associate with the resource share.
        Type: CommaDelimitedList    
Resources:
    ResourceShare:
        Type: AWS::RAM::ResourceShare
        Properties: 
          AllowExternalPrincipals: !Ref AllowExternalPrincipals
          Name: !Ref Name
          Principals: !Ref Principals
          ResourceArns: !Ref ResourceArns
```

Parameters:

```
[
  {
    "ParameterKey": "Name",
    "ParameterValue": "ram-subnets-vpc-sandbox-us-east-2"
  },
  {
    "ParameterKey": "Principals",
    "ParameterValue": ["arn:aws:organizations::123456789012:ou/o-rlcvejgy5g/ou-0v3b-emvezkoj"]                                                                        
  },
  {
    "ParameterKey": "ResourceArns",
    "ParameterValue": ["$[alfred_ssm_/cfct/vpc-sandbox-01-us-east-2/PrivateSubnet1AARN]"
                      ,"$[alfred_ssm_/cfct/vpc-sandbox-01-us-east-2/PrivateSubnet2AARN]"
                      ,"$[alfred_ssm_/cfct/vpc-sandbox-01-us-east-2/PrivateSubnet3AARN]"
                      ,"$[alfred_ssm_/cfct/vpc-sandbox-01-us-east-2/PublicSubnet1ARN]"
                      ,"$[alfred_ssm_/cfct/vpc-sandbox-01-us-east-2/PublicSubnet2ARN]"
                      ,"$[alfred_ssm_/cfct/vpc-sandbox-01-us-east-2/PublicSubnet3ARN]"
    ]
  }
  
]
```

**Comment by groverlalit** [comment]
Thanks for creating a pull request. We have added this our backlog.

**Comment by groverlalit** [comment]
This changes was released in v2.1.0.
